app/src/main/java/pl/futurecollars/invoicing/service/JsonService.java

```diff
@@ -0,0 +1,34 @@
+package pl.futurecollars.invoicing.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import pl.futurecollars.invoicing.model.Invoice;
+
+public class JsonService {
+
+    private final ObjectMapper objectMapper;
+
+    public JsonService() {
+        objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+    }
+
+    public String objectToString(Invoice invoice) {
+        try {
+            return objectMapper.writeValueAsString(invoice) + "\n";
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException("Serialization from object to json string failed", e);
```

Comment:
tu już mamy poprawne przechwytywanie wyjątków - jeśli psuje się coś co uniemożliwia działanie programu to wyrzucamy runtimeException